fabric_cf/orchestrator/core/orchestrator_handler.py

```diff
@@ -132,20 +140,34 @@ def discover_broker_query_model(self, *, controller: ABCMgmtControllerMixin, tok
             self.logger.error(traceback.format_exc())
             raise e
 
-    def list_resources(self, *, token: str, level: int) -> dict:
+    def list_resources(self, *, token: str, level: int, graph_format: str) -> dict:
         """
         List Resources
         :param token Fabric Identity Token
         :param level: level of details (default set to 1)
+        :param graph_format: Graph format
         :raises Raises an exception in case of failure
         :returns Broker Query Model on success
         """
         try:
             controller = self.controller_state.get_management_actor()
             self.logger.debug(f"list_resources invoked controller:{controller}")
 
-            broker_query_model, graph = self.discover_broker_query_model(controller=controller, token=token,
-                                                                         level=level, ignore_validation=True)
+            broker_query_model = None
+            g_format = self.__translate_graph_format(graph_format=graph_format)
+            if g_format == GraphFormat.JSON_NODELINK:
```

Comment:
Or as I noted in slack, instead of checking for format, have an option for level of detail of BQM being requested and for the lowest level that matches what the portal needs, don't check the token (and make sure to cache). This way the lowest level of detail is not timely (as in can be cached, not immediate) and is only an aggregated result.

Reply:
Added the new separate API for portal_resources.